src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.html

```diff
@@ -12,37 +12,41 @@
+                                        {CUSTOM_LABELS.commonPaymentServices}
+                                    </h2>
                                 </lightning-layout-item>
                                 <lightning-layout-item size='6'>
-                                    <template if:true={displayDisableWidgetButton}>
-                                        <template if:false={isReadOnlyMode}>
+                                    <template if:true={isCharge}>
+                                        <template if:false={isEdit}>
                                             <lightning-button variant="base"
-                                                              label={CUSTOM_LABELS.geButtonPaymentDoNotCharge}
-                                                              icon-name='utility:close'
-                                                              icon-position="right"
-                                                              onclick={handleUserDisabledWidget}
-                                                              class='slds-button slds-button_base slds-float_right do-not-charge-card-button'
-                                                              data-qa-locator={qaLocatorDoNotChargeButton}>
+                                                            label={CUSTOM_LABELS.geButtonPaymentDoNotCharge}
+                                                            icon-name='utility:close'
+                                                            icon-position="right"
+                                                            onclick={handleUserDoNotCharge}
+                                                            class='slds-button slds-button_base slds-float_right do-not-charge-card-button'
+                                                            data-qa-locator={qaLocatorDoNotChargeButton}>
                                             </lightning-button>
                                         </template>
                                     </template>
-                                    <template if:true={shouldDisplayEditPaymentInformation}>
-                                        <lightning-button variant="base"
-                                                          label={CUSTOM_LABELS.commonEditPaymentInformation}
-                                                          icon-name='utility:close'
-                                                          icon-position="right"
-                                                          onclick={disableReadOnlyMode}
-                                                          class='slds-button slds-button_base slds-float_right do-not-charge-card-button'
-                                                          data-qa-locator={qaLocatorEditPaymentInformation}>
-                                        </lightning-button>
+                                    <template if:true={isExpiredTransaction}>
```

Comment:
Hey @psadie-sfdo - should this conditional be scoped to include not only "Expired" transactions but also "Authorized" transactions as well? If the answers yes, lmk if I should make this change in my PR or yours. Thanks!